src/yio-interface/entities/mediaplayerinterface.h

```diff
@@ -93,13 +105,25 @@ class MediaPlayerDef : public QObject {
         C_VOLUME_SET,
         C_VOLUME_UP,
         C_VOLUME_DOWN,
+        C_CURSOR_UP,
+        C_CURSOR_DOWN,
+        C_CURSOR_LEFT,
+        C_CURSOR_RIGHT,
+        C_CURSOR_OK,
+        C_MENU,
+        C_CHANNEL_UP,
+        C_CHANNEL_DOWN,
+        C_MUTE_TOGGLE,
```

Comment:
Isn't the mute command usually a toggle function?
Therefore, the existing `C_MUTE` command should be sufficient.

Reply:
Yeah....I can agree.